generate-grpc-stubs.sh

```diff
@@ -21,7 +21,8 @@ set -x
 
 ROOT_DIR=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )" )
 
-python3 -m grpc_tools.protoc -I ./sdv/proto --grpc_python_out=./sdv/proto --python_out=./sdv/proto --mypy_out=./sdv/proto ./sdv/proto/**/*.proto
-mv -f ./sdv/proto/**/*.py ./sdv/proto
-mv -f ./sdv/proto/**/*.pyi ./sdv/proto
-sed -i -e 's/from sdv\..* import/from sdv.proto import/g' -e 's/import sdv\..*\./import sdv.proto./g' ./sdv/proto/*.py*
+python3 -m grpc_tools.protoc -I ./velocitas_sdk/proto --grpc_python_out=./velocitas_sdk/proto --python_out=./velocitas_sdk/proto --mypy_out=./velocitas_sdk/proto ./velocitas_sdk/proto/**/*.proto
+mv -f ./velocitas_sdk/proto/**/*.py ./velocitas_sdk/proto
+mv -f ./velocitas_sdk/proto/**/*.pyi ./velocitas_sdk/proto
+
+sed -i -e 's/from sdv\..* import/from velocitas_sdk.proto import/g' -e 's/import sdv\..*\./import velocitas_sdk.proto./g' ./velocitas_sdk/proto/*.py*
```

Comment:
Hmh, basically this isn't correct: With that we pretend that the data broker stubs belong to the Velocitas SDK, but it isn't the case. Those stubs are only used by Velocitas and belong to data broker.
With this breaking renaming step, we now should move them into the right package/folder structure: `sdv/databroker/v1/...`
With that, we can - in a next step - easily create a Python package to be included (maybe generated on the fly)

Reply:
hmm the path is actually coming from kuksa: https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker/proto/sdv/databroker/v1
Thats why I left it as it is...